cats/cats-redis/src/main/java/com/netflix/spinnaker/cats/redis/JedisClientDelegate.java

```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright 2017 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.netflix.spinnaker.cats.redis;
+
+import redis.clients.jedis.BinaryJedisCommands;
+import redis.clients.jedis.Jedis;
+import redis.clients.jedis.JedisCommands;
+import redis.clients.jedis.MultiKeyCommands;
+import redis.clients.jedis.Pipeline;
+import redis.clients.jedis.commands.RedisPipeline;
+
+import java.util.function.Consumer;
+import java.util.function.Function;
+
+public class JedisClientDelegate implements RedisClientDelegate {
+
+  JedisSource jedisSource;
+
+  public JedisClientDelegate(JedisSource jedisSource) {
```

Comment:
if this is the only place still using JedisSource you could probably just switch this to Pool<Jedis> here